src/components/Confirm/AddAssetConfirm/AddAssetConfirm.vue

```diff
@@ -1,48 +1,106 @@
+        <v-flex xs12>
+          <v-layout mx-n2>
+            <v-flex xs6 px-2>
+              <v-btn block large text @click="rejectAddToken">{{ t('homeToken.cancel') }}</v-btn>
+            </v-flex>
+            <v-flex xs6 px-2>
+              <v-btn block large color="torusBrand1" class="white--text" type="button" @click="approveAddToken">
+                {{ t('homeToken.addToken') }}
+              </v-btn>
+            </v-flex>
+          </v-layout>
+        </v-flex>
+      </v-layout>
+    </v-card>
+  </div>
+  <div v-else>
     <v-layout class="card-header" wrap>
       <v-flex text-center xs12 py-10 px-6>
-        <div class="display-1">{{ t('homeToken.addTokens') }}</div>
+        <div class="display-1">{{ t('homeAssets.add') }}</div>
+        <v-btn class="close-btn" icon aria-label="Close Add Token" title="Close Add Token" @click="rejectAddNft">
+          <v-icon>$vuetify.icons.close</v-icon>
+        </v-btn>
       </v-flex>
     </v-layout>
-    <v-layout mx-6 pt-6 pb-4 wrap>
-      <v-flex xs12>
-        <div class="title">{{ t('homeToken.likeToAddToken') }}</div>
+    <v-layout mx-7 pt-6 pb-4 wrap>
+      <v-flex xs12 class="text-center">
+        <div class="title">{{ t('homeAssets.infoTitle') }}</div>
       </v-flex>
     </v-layout>
     <v-divider></v-divider>
-    <v-layout mb-8 mx-6 pt-6 wrap class="align-center">
-      <v-flex xs8 mb-3>
-        <div class="body-2 font-weight-bold">{{ t('homeToken.token') }}</div>
-      </v-flex>
-      <v-flex xs4 text-right mb-3>
-        <div class="body-2 font-weight-bold">{{ t('homeToken.balance') }}</div>
-      </v-flex>
-      <v-flex xs8>
-        <div class="d-flex align-center">
-          <img :src="`${image}`" class="inline-small d-inline-flex" height="36" />
-          <div class="ml-2 body-1">{{ name }}</div>
+    <v-layout mb-8 mx-7 pt-6 wrap class="align-center">
+      <v-flex xs12>
+        <div class="body-2 font-weight-bold">{{ t('homeAssets.infoCollectible') }}</div>
+        <div class="d-flex mb-5">
+          <div>
+            <img
+              :src="image"
+              style="width: 136px; height: auto"
+              onerror="if (!this.src.includes('/images/nft-placeholder.svg')) this.src = '/images/nft-placeholder.svg';"
+            />
+          </div>
+          <div class="ml-auto flex-shrink-1" style="width: 136px">
+            <div class="body-2 text_1--text mb-3">{{ name }}</div>
+            <div class="caption text_1--text mb-1">{{ t('homeAssets.infoId') }}</div>
+            <div class="caption text_3--text mb-2">{{ id }}</div>
+            <div class="caption text_1--text mb-1">{{ t('homeAssets.infoContractAdd') }}</div>
+            <div class="caption text_3--text">{{ address }}</div>
+          </div>
+        </div>
+        <div class="caption text_1--text mb-3">{{ t('homeAssets.infoExplorer') }}</div>
+        <div class="caption text_3--text mb-4">https://etherscan.io/address/{{ address }}</div>
```

Comment:
this won't be always etherscan, it should be decided on type of network. There is some function which already derives etherscan url based on network